### PR TITLE
[Dashboard] Delete old dashboard

### PR DESCRIPTION
**Pull request by rkooo567**
## Why are these changes needed?

Deleting the old dashboard from our repo as they are not needed anymore & we cannot turn it on again anyway.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by edoakes — APPROVED** [review]

**Comment by rkooo567** [comment]
Will merge the PR after @mfitton approves it.

**Review by mfitton — APPROVED** [review]
Sweet, yeah this is great @rkooo567
Next up, renaming all the `new_dashboard` stuff to plain old `dashboard`, moving the `dashboard` directory from top level into the python dir, and removing the sym link from `dashboard` to the `new_dashboard` directory.

**Pull request merged**
